template/client/store/mutations.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import type {
+  CommentType,
+  CommentPayloadType,
+  StateType,
+  RawCommentType
+} from '~/types'
+
+const SET_COMMENTS = 'root/SET_COMMENTS'
```

Comment:
Defining mutations types inside mutations is not common for us.
This is a common practice to create `store/types.js` with exporting these constants.

This way it is easy to import them everywhere.